src/definition/accessors/IAppUpdateContext.ts

```diff
@@ -0,0 +1,6 @@
+import type { IUser } from '../users';
+
+export interface IAppUpdateContext {
+    user: IUser | null;
```

Comment:
This looks like an optional property, is there a reason why you didn't use `user?: IUser`?

Reply:
Hmmm, good question. Cannot remember 🤔 let me double check

Reply:
```suggestion
    user?: IUser;
```